Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/OS - Hybrid Sleep Disabled.json

```diff
@@ -8,7 +8,7 @@
       }
     }
   ],
-  "description": "Disables hybrid sleep by setting hibernate timeout settings to 0. Hiberfile is not generated with these settings.",
+  "description": "What is it: Disables hybrid sleep by setting hibernate timeout settings to 0. Hiberfile is not generated with these settings. ",
```

Comment:
**What does this do?**
Disables hybrid sleep by setting hibernate timeout settings to 0. Hiberfil is not generated with these settings. 

**Why should you use this?**


**What is the end-user impact?**


**Learn more**
N/A

@metamenos this seems simple, but I'm really not sure what this one means even after checking the portal...also hiberfile or hiberfil?

Reply:
We are going to keep this policy to turn off hybrid joins.

Reply:
@metamenos why? -jeff

Reply:
So you want to remove hybrid sleep, I thought in the teams channel you thought we should remove it? I thought no one liked hybrid sleep.